docs/serve_ipex-llm.md

```diff
@@ -0,0 +1,22 @@
+## Deploying and Serving LLMs with ipex-LLM
+[ipex-LLM](https://ipex-llm.readthedocs.io/en/latest/doc/LLM/index.html) is a library for running LLM (large language model) on Intel XPU (from Laptop to GPU to Cloud) using INT4 with very low latency (for any PyTorch model).
```

Comment:
Pls honor the original capitalization. ipex-LLM should be IPEX-LLM. See: 
https://ipex-llm.readthedocs.io/en/latest/doc/LLM/index.html
Pls update other cases as well.